src/UI/Component/TriggeredSignal.php

```diff
@@ -0,0 +1,33 @@
+<?php
+namespace ILIAS\UI\Component;
+
+/**
+ * Interface TriggeredSignal
+ *
+ * Describes a signal that is triggered by a component on an event
```

Comment:
Unfortunately, I do not understand this. Why do we need this? Where/when would it be used? Currently, I would need to check in the implementation.